pkg/virtctl/console/console.go

```diff
@@ -131,3 +139,70 @@ func (c *Console) Run(args []string) error {
 	}
 	return nil
 }
+
+// AttachConsole attaches stdin and stdout to the console
+// in -> stdinWriter | stdinReader -> console
+// out <- stdoutReader | stdoutWriter <- console
+func AttachConsole(stdinReader, stdoutReader *io.PipeReader, stdinWriter, stdoutWriter *io.PipeWriter, message string, resChan <-chan error) (err error) {
```

Comment:
Consider renaming it to `Attach`. Flows better when using it as import from somewhere else (`console.Attach`).